oleesir/banking-app
# #164730283User should be able to see the activate or deactivate user account page.

#### What does this PR do?
implement activate-deactivate-template
#### Description of Task to be completed?
implement activate-deactivate-template
#### How should this be manually tested?
* Navigate to the UI folder and click on bank_accounts.html in your browser
* You will see an activate and activate link
#### What are the relevant pivotal tracker stories?
[#164730283](https://www.pivotaltracker.com/story/show/164730283)
#### Any background context you want to add?
N/A
#### Screenshots
<img width="1280" alt="Screenshot 2019-03-29 at 5 12 49 AM" src="https://user-images.githubusercontent.com/17679511/55209142-607ff200-51e1-11e9-95d1-1730faaf7025.png">
<img width="1280" alt="Screenshot 2019-03-29 at 5 12 42 AM" src="https://user-images.githubusercontent.com/17679511/55209179-8a391900-51e1-11e9-9696-7c09100b36dc.png">
